src/templates/pycontw-2020/contents/zh/venue.html

```diff
@@ -0,0 +1,109 @@
+{% extends 'contents/_venue.html' %}
+
+{% block title %}會場{% endblock title %}
+
+{% block map_info %}
+
+<h2>
+	<span>國立成功大學</span>
+	<span>國際會議廳</span>
+</h2>
+<p>701 台南市東區大學路 1 號</p>
+
+<script>window.VENUE_NAME = '國立成功大學 國際會議廳'</script>
+
+{% endblock map_info %}
+
+{% block content %}
+
+<h1>會場</h1>
+
+{{ block.super }}
+
+<h2>交通方式</h2>
+
+<h3>桃園機場（TPE）出發</h3>
+
+<h4>1. 桃園機場（TPE）到桃園高鐵</h4>
+
+<ul>
+  <li>
+    捷運
+    <ul>
+      <li>機場航廈站（A12 or A13）到高鐵桃園站（A18）</li>
+      <li><a href="https://www.tymetro.com.tw/tymetro-new/tw/_pages/travel-guide/timetable-search.php">時刻表</a></li>
+      <li>票價：NT$ 25</li>
+    </ul>
+  </li>
+  <li>
+    Uber
+    <ul>
+      <li>UberX Airport（NT$ 1,000 - 1,200）</li>
+      <li>UberXL Airport（NT$ 1,250 - 1,500）</li>
+      <li>UberBlack Airport（NT$ 1,500 - 1,800）</li>
+    </ul>
+  </li>
+</ul>
+
+<h4>2. 桃園高鐵站到台南高鐵站</h4>
+
+<ul>
+  <li>票價：NT$ 1,190</li>
+  <li><a href="https://www.thsrc.com.tw/tw/TimeTable/SearchResult">時刻表</a></li>
+</ul>
+
+<h4>3. 台鐵沙崙站到台南火車站</h4>
+
+<ul>
+  <li>沙崙站（4272）到台南站（4220）</li>
```

Comment:
It would be better if the item of each step is consistent. I would suggest removing this line and add the station number to the title.  You can also move the fare to the front of schedule.

Reply:
Fixed it. Thank you.